.github/workflows/ci.yml

```
@@ -31,4 +31,17 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           slug: Mrityunjoy99/sample-go
+          files: ./coverage.out
+          flags: unittests
+          fail_ci_if_error: true
+      - name: Upload test results to Codecov
+        if: ${{ !cancelled() }}
+        uses: codecov/test-results-action@v1
+        with:
+            token: ${{ secrets.CODECOV_TOKEN }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)